spec/datadog/core/telemetry/heartbeat_spec.rb

```diff
@@ -6,7 +6,7 @@
   subject(:heartbeat) { described_class.new(enabled: enabled, interval: interval, &block) }
```

Comment:
Minor: I suggest propagating the `_seconds` prefix internally as well:

```suggestion
  subject(:heartbeat) { described_class.new(enabled: enabled, heartbeat_interval_seconds: heartbeat_interval_seconds, &block) }
```